docs/ens.md

```diff
@@ -0,0 +1,104 @@
+---
+id: ens
+title: Adding accounts to an ENS domain
+sidebar_label: Using ENS with DOT/KSM accounts
+---
+
+ENS (Ethereum Name Service) is a system of smart contracts on the Ethereum blockchain which allows
+users to claim domain names like `bruno.eth`. Supporting wallets can then allow senders to input ENS
+domains instead of long and unwieldy addresses. This prevents phishing, fraud, typos, and adds a
+layer of usability on top of the regular wallet user experience.
```

Comment:
Nice intro, but could you add in why might someone be incentivized to use ens with kusama and polkadot?

Reply:
Are the "use domains instead of unwieldy addresses" and "prevents phishing.." sentences not a selling point?

Reply:
OH at first I wasn't understanding what the purpose of ens was, now I do. This makes total sense now